src/bicep/examples/newWorkload/newWorkload.bicep

```diff
@@ -40,7 +42,7 @@ param subnetName string = '${workloadName}-subnet'
 param subnetAddressPrefix string = '10.0.125.0/27'
 param subnetServiceEndpoints array = []
 
-param logStorageAccountName string = toLower(take('logs${uniqueString(workloadName)}', 24))
+param logStorageAccountName string = toLower(take('logs${uniqueString(subscription().subscriptionId, workloadName)}', 24))
```

Comment:
We have a new naming convention for storage accounts described like this: https://github.com/Azure/missionlz/blob/50d7c1e4892014faa5e7ec4edf3bb14ef5934d7c/src/bicep/mlz.bicep#L482

I don't think that's a tax we should have to pay in this change, but, I do see how making a "naming" module that accepts some params and outputs names by convention could be helpful, especially if we intend to pollinate the naming convention to the add-ons templates. I'll jot down some thoughts in an enhancement Issue.

+@brooke-hamilton for awareness.